adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Exceptions/AdalError.cs

```diff
@@ -295,5 +295,26 @@ public static class AdalError
         /// Broker redirectUri has incorrect format
         /// </summary>
         public const string BrokerRedirectUriIncorrectFormat = "broker_redirect_uri_incorrect_format";
+
+        /// <summary>
+        /// Cannot fetch the upn from the OS, possibly due to missing permissions
+        /// </summary>
+        public const string CannotAccessUserInformationOrUserNotDomainJoined = "user_information_access_failed";
+
+        /// <summary>
+        /// Error code that denotes that this library is still polling to see if the user 
```

Comment:
This is for information (not an error per se).

Reply:
imo. it still makes sense. Good addition to the text. perhaps it should be a remark section? and it could say Informational ?

---
In reply to: [226308298](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1333#discussion_r226308298) [](ancestors = 226308298)